File: lib/plugins/yourface.js
module.exports = function(bot, config) {

  bot.respond(/your face is (.*)$/i, function(msg, reply) {
    reply("Oh yeah... well, your face is...");
    reply("!face " + msg.from);
  });

  bot.help("yourface", function(msg, reply) {
    reply("That's right!");
  });

};

File: plugins/yourface.py
class Plugin:
    def __call__(self, bot):
        bot.on_respond(r"your face is .+$", self.on_respond)
        bot.on_help("yourface", self.on_help)

    def on_respond(self, bot, msg, reply):
        reply("Oh yeah… well, your face is…")
        reply("!face " + msg["sender"])

    def on_help(self, bot, msg, reply):
        reply("That's right!")
